requirements/hpu.txt

```diff
@@ -7,4 +7,8 @@ ray
 triton==3.1.0
 setuptools>=77.0.3
 setuptools-scm>=8
-vllm-hpu-extension @ git+https://github.com/HabanaAI/vllm-hpu-extension.git@22abb7a
+vllm-hpu-extension @ git+https://github.com/HabanaAI/vllm-hpu-extension.git@3ab2f63
+
```

Comment:
This commit id should change to the corresponding one in 1.22.0_next 
https://github.com/HabanaAI/vllm-hpu-extension/pull/302/commits (pick latest commit id) and then have gaudi test run. 
Once hpu-extension merged, then update the commit id again with official one .